app/components/dashboard/hypertension/overdue_patients_called_component.html.erb

```diff
@@ -0,0 +1,49 @@
+<!--Implementation Pending - Overdue Patients Graph-->
+<div id="overdue-patients-called-trend" class="d-lg-flex w-lg-50 pl-lg-2">
+  <%= render Dashboard::Card::GraphComponent.new(id: "overdueCalledTrend", data: graph_data, period: @period) do |c| %>
+    <%= c.title(title: "Overdue patients called".html_safe, subtitle: t("overdue_patients_called.reports_card_subtitle")) do |title| %>
+        <%= title.tooltip({ "Numerator" => t("overdue_patients_called.numerator"),
```

Comment:
I think we can refactor the tooltip component to 
1. Take a list of maps, the individual maps can be separated by a divider, this way we don't have to pass `divider => ""`
2. Change the values of the hash to something like { text: "...", italics: true, .. }.

Reply:
This might be a large change for this PR.
We can either revert the changes on these components and stick to a simple tooltip without the styling or refactor this in another PR
@jamiecarter7

Reply:
Don't need to block the, release for this, the refactoring can be done after the release. Let's create a story in the is. epic to do the refactoring though.